src/popup/components/buttons/BtnClose.vue

```diff
@@ -4,7 +4,7 @@
     v-bind="$attrs"
     @click="$emit('click', $event)"
   >
-    <CloseIcon />
+    <CloseIcon class="icon" />
```

Comment:
This class is not used within the scope of this component. I guess it can be removed.